src/server/db_setup.ts

```diff
@@ -56,8 +56,6 @@ async function init(): Promise<void> {
             csvData.push(data);
         })
         .on("end", function() {
-            // remove the first line: header
-            csvData.shift();
```

Comment:
What changed such that this isn't needed? Does fast-csv skip/understand that the first line is usually a header, so this basically changes nothing?

Reply:
The csvData.shift() was removing the first line of the GameLogs.csv file. This line was info for one of the players in the first game - it was (20200911a,1,Shelby,53). This was messing up the data for two player and three player games. Removing this line fixed that issue.